apps/meteor/client/components/WarningModal.tsx

```diff
@@ -1,21 +1,21 @@
 import { Button, Modal } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
-import type { ReactElement } from 'react';
+import type { ReactElement, ReactNode } from 'react';
 import React from 'react';
 
 type WarningModalProps = {
-	text: string;
-	confirmText: string;
+	text: ReactNode;
+	confirmText: ReactNode;
+	cancelText?: ReactNode;
+	confirm: () => void;
+	cancel?: () => void;
 	close: () => void;
-	cancel: () => void;
-	cancelText: string;
-	confirm: () => Promise<void>;
 };
 
 const WarningModal = ({ text, confirmText, close, cancel, cancelText, confirm, ...props }: WarningModalProps): ReactElement => {
 	const t = useTranslation();
 	return (
-		<Modal {...props}>
+		<Modal open {...props}>
```

Comment:
why?

Reply:
Dialog element need the open prop to expose their content (aria-hidden): https://testing-library.com/docs/queries/byrole/#hidden